### ISSUE TITLE
implementing a Music player using data structures in c++

### ISSUE THREAD

**Issue by sanjana317** [hitesh-3/simple-coding#2]
Hey, I would like to add a mini project. The idea is to implement data structures and create a basic music player in order to understand the concepts of data structures better.  Data structures that will be used are doubly linked lists, stacks, and queues.

**Comment by AkshatGoel-321** [comment]
sure that's quite an interesting project. you may do so. I am assigning this to you.

**Comment by AkshatGoel-321** [comment]
That's quite an interesting project. you may do so.

**Comment by hitesh-3** [comment]
yes it's quite an interesting project. You may do so. Try not to use complex code

**Comment by sanjana317** [comment]
Okay, thank you.
